docs/user-guide/deploying-application/README.md

```diff
@@ -1,10 +1,12 @@
 # Deploying Application
 
-Welcome! This is the documentation for Deploying Applications
+Welcome! This is the documentation for Deploying Applications.
 
 **Parts of Documentation**
```

Comment:
Should we change this to `Contents` or something else?

Reply:
Done